When I tried to run the new test script, I got the following output:

``` python
test_authenticate (__main__.PoodleDoTest) ... ok
test_getAccountInfo (__main__.PoodleDoTest) ... ok
test_getToken (__main__.PoodleDoTest) ... ok
test_getUserid (__main__.PoodleDoTest) ... ERROR

======================================================================
ERROR: test_getUserid (__main__.PoodleDoTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./test_poodledo.py", line 79, in test_getUserid
    self.assertRaises(ToodledoError, api.getUserid, self.user_email,self.password + 'wrong_password')
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/case.py", line 456, in assertRaises
    callableObj(*args, **kwargs)
  File "/Users/acompton/src/poodledo-pull/apiclient.py", line 184, in getUserid
    email=email, pass_=passwd).text
  File "/Users/acompton/src/poodledo-pull/apiclient.py", line 124, in _call
    stream = self._urlopener.open(url)
  File "./test_poodledo.py", line 23, in open
    return open(self.url_map[url],'r')
IOError: [Errno 2] No such file or directory: 'testdata/getUserid_bad.xml'

----------------------------------------------------------------------
Ran 4 tests in 0.012s
```

I don't see this file in your fork. Is it supposed to not be there?

--- a/testdata/getUserid_bad.xml
+++ b/testdata/getUserid_bad.xml
@@ -1,0 +1,1 @@
+<?xml version="1.0" encoding="UTF-8"?><error>Bad Password</error>

